compiler-plugin-tests/src/test/resources/diagnostics/sample5/main.bal

```diff
@@ -1,29 +0,0 @@
-// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
-//
-// WSO2 LLC. licenses this file to you under the Apache License,
-// Version 2.0 (the "License"); you may not use this file except
-// in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-import ballerinax/mssql;
-
-public function main() returns error? {
-    mssql:Client dbClient = check new();
-    _ = check dbClient->query(``);
-    _ = check dbClient->queryRow(``);
-    check invokeQuery(dbClient);
-    check dbClient.close();
-}
-
-function invokeQuery(mssql:Client dbClient) returns error? {
-    _ = check dbClient->query(``);
-}
```

Comment:
why we delete these two files? Are we not using them?

Reply:
Yes. This sample is not used anywhere.